packages/hub/node-tests/services/card-space-validator-test.ts

```diff
@@ -4,40 +4,16 @@ import { setupHub } from '../helpers/server';
 describe('CardSpaceValidator', function () {
   let { getContainer } = setupHub(this);
 
-  let invalidUrls = [
-    'www.card.space',
```

Comment:
Should we add a check for these the subdomains to the slug validator in the merchant info?

Reply:
the merchant info slug tests check for [forbidden words](https://github.com/cardstack/cardstack/blob/aaffdbc4142fba53032ca1662b137a4c741673a9/packages/hub/node-tests/routes/merchant-infos-test.ts#L181) and [only lowercase letters](https://github.com/cardstack/cardstack/blob/aaffdbc4142fba53032ca1662b137a4c741673a9/packages/hub/node-tests/routes/merchant-infos-test.ts#L123) which I think covers everything?
*